guides/drivers.rst

```diff
@@ -32,23 +33,23 @@ cannot evaluate JavaScript and Selenium2Driver cannot get the response status
 code. In such cases, the driver will always throw a meaningful
 ``Behat\Mink\Exception\UnsupportedDriverActionException``.
 
-======================  =================  =========  ======  ========  ====
-Feature                 BrowserKit/Goutte  Selenium2  Zombie  Selenium  Sahi
-======================  =================  =========  ======  ========  ====
-Page traversing         Yes                Yes        Yes     Yes       Yes
-Form manipulation       Yes                Yes        Yes     Yes       Yes
-HTTP Basic auth         Yes                No         Yes     No        No
-Windows management      No                 Yes        No      Yes       Yes
-iFrames management      No                 Yes        No      Yes       No
-Request headers access  Yes                No         Yes     No        No
-Response headers        Yes                No         Yes     No        No
-Cookie manipulation     Yes                Yes        Yes     Yes       Yes
-Status code access      Yes                No         Yes     No        No
-Mouse manipulation      No                 Yes        Yes     Yes       Yes
-Drag'n Drop             No                 Yes        No      Yes       Yes
-Keyboard actions        No                 Yes        Yes     Yes       Yes
-Element visibility      No                 Yes        No      Yes       Yes
-JS evaluation           No                 Yes        Yes     Yes       Yes
-Window resizing         No                 Yes        No      No        No
-Window maximizing       No                 Yes        No      Yes       No
-======================  =================  =========  ======  ========  ====
+======================  =================  =========  ======  ======  ========  ====
```

Comment:
1. Are you sure, that `ChromeDriver` has `Yes` everywhere?
2. Does that apply also to headless mode?

Reply:
I'm sure.

It implements the entire CoreDriver API. The driver tests suite passes with and without --headless https://gitlab.com/DMore/chrome-mink-driver/pipelines/23431065

Headless mode doesn't really have limitations. It even supports screenshots.